tasks/.env

```diff
@@ -1,3 +1,4 @@
 NEXT_PUBLIC_API_URL=http://localhost:8080
 NEXT_PUBLIC_MOCK=true
 NEXT_PUBLIC_REQUEST_LOGGING=false
+NEXT_PUBLIC_OAUTH_REDIRECT_URI=http://localhost:3000/auth/callback
```

Comment:
Changes to this file don't seem to be part of the issue

Reply:
#246 @DasProffi